kratos/python_scripts/assign_scalar_variable_process.py

```diff
@@ -74,7 +72,7 @@ def __init__(self, Model, settings ):
             raise Exception(msg)
 
         self.model_part = Model[settings["model_part_name"].GetString()]
-        self.mesh = self.model_part.GetMesh(settings["mesh_id"].GetInt())
```

Comment:
I think we can even remove `self.mesh` if we pass the nodes container as `self.model_part.Nodes` below.